crates/sozo/src/commands/auth.rs

```diff
@@ -22,7 +23,7 @@ pub enum AuthCommand {
         model: String,
 
         #[arg(help = "Name of the system to grant writer access to.")]
-        system: String,
+        system: FieldElement,
```

Comment:
Let's rename system here and elsewhere to contract

Reply:
done !